config/settings.yml

```diff
@@ -112,6 +112,7 @@ feature:
   # results file notifications requires that my_files be on as well
   results_file_notifications_on: true
   set_statement_search_start_date_on: false
+  skip_statement_emails_on: false
```

Comment:
So I did this because I believe the default should be to send them. Turning a feature on feels like overriding a default. But maybe I'm thinking about it the wrong way.